x-pack/plugins/maps/common/constants.ts

```diff
@@ -63,6 +63,7 @@ export enum SOURCE_TYPES {
   ES_SEARCH = 'ES_SEARCH',
   ES_PEW_PEW = 'ES_PEW_PEW',
   ES_TERM_SOURCE = 'ES_TERM_SOURCE',
+  ES_ML_ANOMALIES = 'ML_ANOMALIES',
```

Comment:
Would it be better to import the `ML_ANOMALIES` const from x-pack/plugins/ml/public/maps/anomaly_source_factory.ts? I was expecting to use the const for [this check](https://github.com/elastic/kibana/pull/136345/files#diff-cc4aec037353ddc28b80e66cca4f1ff518da10f1d08f933eb332a8421be266bdR198-R201).

Reply:
Chatted offline. Its to bad there is no way for a plugin to update telemetry. Since maps plugin has to track telemetry, constant should stay in maps plugin. Nothing to change.